requirements/requirements.in

```diff
@@ -23,7 +23,6 @@ psycopg2
 django-ses
 
 # security and identity system
-social-auth-app-django
```

Comment:
Please follow the instructions in the [requirements README](https://github.com/CMSgov/bluebutton-web-server/tree/master/requirements) to update the `vendor` directory. You may want to start with `rm -Rf vendor/*` since we're subtracting reqs here.